page-what-we-do.php

```diff
@@ -0,0 +1,116 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+
+<?php
+if( have_posts() ) {
+  while( have_posts() ) {
+    the_post();
+
+    $bios = get_post_meta($post->ID, '_igv_bios_group', true);
+
+    $realestate_text = get_post_meta($post->ID, '_igv_realestate_text', true);
+    $realestate_images = get_post_meta($post->ID, '_igv_realestate_images', true);
+
+    $community_text = get_post_meta($post->ID, '_igv_community_text', true);
+    $community_images = get_post_meta($post->ID, '_igv_community_images', true);
+
+    $philanthropy_text = get_post_meta($post->ID, '_igv_philanthropy_text', true);
+    $philanthropy_images = get_post_meta($post->ID, '_igv_philanthropy_images', true);
+
+
+?>
+
+        <article <?php post_class('grid-item item-s-12'); ?> id="post-<?php the_ID(); ?>">
+          <section id="what-we-do-hero">
+            <div class="container">
+              <div class="grid-row">
+                <div class="grid-item item-s-12">
+                  <?php the_content(); ?>
+                </div>
+              </div>
+            </div>
+          </section>
+
+<?php
+    if (has_post_thumbnail()) {
+?>
+
+          <section id="what-we-do-splash" class="splash-with-image" style="background-image: url(<?php echo get_the_post_thumbnail_url($post->ID, 'splash'); ?>)">
+
+          </section>
+
+<?php
+    }
+
+    if (!empty($bios)) {
+?>
+          <section id="what-we-do-bios">
+            <div class="container">
+<?php
+      foreach ($bios as $bio) {
+?>
+              <div class="grid-row">
+                <div class="grid-item item-s-12 item-m-4">
+                  <div class="highlight-section-title">
+                    <h3 class="font-heavy"><?php echo !empty($bio['name']) ? $bio['name'] : ''; ?></h3>
```

Comment:
should the condition be outside the `<h3>` so it can not render anything if empty?